layouts/community/ortho_4x12/bredfield/keymap.c

```diff
@@ -0,0 +1,181 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Keymap layers
```

Comment:
Recommended but optional: using an `enum` for your layer indexes.

This is programmatically equivalent to your current definitions:

```c
// Keymap layers
enum layer_names {
    _BASE_LAYER,
    _RAISE_LAYER,
    _LOWER_LAYER,
    _NUMPAD_LAYER,
    _NAVIGATION_LAYER,
    _UI_LAYER,
    _KEYBOARD_LAYER,
};
```

Reply:
For this number of layers, I agree.  It will be a lot easier to manage, as an enum